services/autorust/codegen/tests/azure_rest_api_specs.rs

```diff
@@ -9,8 +9,9 @@ use std::path::PathBuf;
 
 type Result<T> = std::result::Result<T, Box<dyn std::error::Error>>;
 
-const COMMON_TYPES_SPEC: &str = "../../azure-rest-api-specs/specification/security/resource-manager/common/v1/types.json";
-const VMWARE_SPEC: &str = "../../azure-rest-api-specs/specification/vmware/resource-manager/Microsoft.AVS/stable/2020-03-20/vmware.json";
+const COMMON_TYPES_SPEC: &str = "../../../../azure-rest-api-specs/specification/security/resource-manager/common/v1/types.json";
```

Comment:
With as many paths here for build time, would it be better to define variables for roots? We do this in a lot of other languages. Could do something like:

```rust
const COMMON_TYPES_SPEC: &str = concat!(env!("REPO_ROOT"), "...");
```

And define "cargo:rustc-env=REPO_ROOT=..." in a build.rs or something.

IMO, much easier to get right and not have to iterate if, for example, you miscounted how many directories you need to go up.

Just a thought.

Reply:
It looked into making it configurable, but then reverted. This change is due to the move this repo.